src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -425,6 +435,123 @@ public void ValidateAndSetIgnoreUnexpectedElementsAndAttributesFromString(string
     this.IgnoreUnexpectedElementsAndAttributes = boolValue;
 }
 
+public void ValidateAndSetCustomHttpHeadersFromString(string stringValue) 
+{
+    this.CustomHttpHeaders = new Dictionary <string, string> ();

```

Comment:
pay attention to the stylecop. for example, remove unused whitespace after Dictionary

Reply:
Done